packages/css/tests/analysis/analyzer.test.ts

```diff
@@ -93,6 +93,68 @@ describe('analyzeCSS', () => {
   });
 });
 
+describe('should emit correctly located issues regardless of invisible characters', () => {
+  const testCases: Array<
+    [type: 'single' | 'multiple', expectation: [line: number, column: number]]
+  > = [
+    ['single', [5, 2]],
+    ['multiple', [7, 4]],
+  ];
```

Comment:
I am not unable to checkout your branch to try my suggestion for some reason, but couldn't we just write:

```suggestion
  const testCases = [
    ['single', [5, 2]],
    ['multiple', [7, 4]],
  ] as const;
```